pkg/cmd/auth-apiserver/cmd.go

```diff
@@ -3,14 +3,14 @@ package oauth_apiserver
 import (
 	"io"
 
+	"github.com/spf13/cobra"
```

Comment:
nit: add an empty space

Reply:
How many import section do we want? 5? 6? :)

Reply:
we usually have:
```
std

other pkgs

kube

openshift
```